packages/builders/scripts/inter-protocol/price-feed-core.js

```diff
@@ -67,38 +68,49 @@ export const defaultProposalBuilder = async (
   });
 };
 
-export const createGov = async (homeP, endowments) => {
+export const getBrandsAndProposalBuilder = async (options, endowments) => {
   const { lookup } = endowments;
 
+  await null;
+  const options1 = {
+    ...options,
+    brandIn: await lookup(options.IN_BRAND_LOOKUP).catch(() => undefined),
+    brandOut: await lookup(options.OUT_BRAND_LOOKUP).catch(() => undefined),
+  };
+  return powers => defaultProposalBuilder(powers, options1);
+};
+
+export const createGov = async (homeP, endowments) => {
+  const { writeCoreProposal } = await makeHelpers(homeP, endowments);
+
   const {
     AGORIC_INSTANCE_NAME,
     IN_BRAND_DECIMALS = '6',
-    IN_BRAND_LOOKUP = JSON.stringify(['wallet', 'brand', 'BLD']),
+    IN_BRAND_LOOKUP = ['wallet', 'brand', 'BLD'],
     OUT_BRAND_DECIMALS = '6',
-    OUT_BRAND_LOOKUP = JSON.stringify(['agoricNames', 'oracleBrand', 'USD']),
+    OUT_BRAND_LOOKUP = ['agoricNames', 'oracleBrand', 'USD'],
     ORACLE_ADDRESSES,
   } = process.env;
-
   assert(AGORIC_INSTANCE_NAME, 'AGORIC_INSTANCE_NAME is required');
   assert(ORACLE_ADDRESSES, 'ORACLE_ADDRESSES is required');
-
   const oracleAddresses = ORACLE_ADDRESSES.split(',');
 
-  const { writeCoreProposal } = await makeHelpers(homeP, endowments);
+  const options = {
+    AGORIC_INSTANCE_NAME,
+    IN_BRAND_DECIMALS: parseInt(IN_BRAND_DECIMALS, 10),
+    OUT_BRAND_DECIMALS: parseInt(OUT_BRAND_DECIMALS, 10),
+    IN_BRAND_LOOKUP,
+    OUT_BRAND_LOOKUP,
+    oracleAddresses,
+  };
 
-  const inLookup = JSON.parse(IN_BRAND_LOOKUP);
```

Comment:
I think this is a regression w.r.t. CLI usage.

@michaelfig I wonder if we have any tests for that.